Sources/StreamChat/Workers/ChannelUpdater.swift

```diff
@@ -918,19 +921,12 @@ extension ChannelUpdater {
     ) async throws -> ChannelPayload {
         // Just populate the closure since we select the endpoint based on it.
         let useCreateEndpoint: ((ChannelId) -> Void)? = channelQuery.cid == nil ? { _ in } : nil
-        let parameter = channelQuery.pagination?.parameter
-        let reset = parameter == nil || parameter?.isJumpingToMessage == true
-        let actions = ChannelUpdateActions(
-            memberListSorting: memberSorting,
-            resetMembers: reset,
-            resetWatchers: reset
-        )
```

Comment:
Moved the fix I used for state layer to the completion handler based update method instead. State layer already has tests for this case.